lightning/src/ln/features.rs

```diff
@@ -101,6 +101,8 @@ mod sealed {
 			StaticRemoteKey,
 			// Byte 2
 			,
+			// Byte 3
+			ShutdownAnySegwit,
```

Comment:
See my point above but I think for now it's better to keep this new feature optional.